src/braket/circuits/noise.py

```diff
@@ -87,34 +91,43 @@ def register_noise(cls, noise: "Noise"):
         setattr(cls, noise.__name__, noise)
 
 
-class SingleProbabilisticNoise(Noise):
+class SingleProbabilisticNoise(Noise, Parameterizable):
     """
     Class `SingleProbabilisticNoise` represents the bit/phase flip noise channel on N qubits
     parameterized by a single probability.
     """
 
     def __init__(
-        self, probability: float, qubit_count: Optional[int], ascii_symbols: Sequence[str]
+        self,
+        probability: Union[FreeParameterExpression, float],
+        qubit_count: Optional[int],
+        ascii_symbols: Sequence[str],
+        max_probability: float = 0.5,
```

Comment:
Does this value need to have a ceiling check?

Reply:
Probably, but that's probably best scoped for a different change.